core/src/main/scripts/import_data_validator/validateData.py

```diff
@@ -567,7 +566,10 @@ def validate(self):
                 return
 
             # parse the start-of-file comment lines
-            self.processTopLines(top_comments)
+            if not self.processTopLines(top_comments):
+                self.logger.info(
+                    'Invalid header comments, skipped data in file')
```

Comment:
is this not related to the clinical data file? Maybe it should be moved to this specific validator instead.